beater/sophoscentralbeat.go

```diff
@@ -166,11 +171,14 @@ func GetSophosEvents(scb Sophoscentralbeat) (int64, error) {
 		scb.logger.Info("Events sent")
 	}
 
+	counterLock.Lock()
 	allEventsLogs = receivedEventsLogCount

```

Comment:
@rajanjoshi18  Instead, directly use `logsReceivedInCycle` just near to publishing received logs.. It's better to create the function like `updateCycleLogCounter` and call it from alert and events function. Make sure to lock the counter update